modules/develop/pages/manage-topics/cloud-topics.adoc

```diff
@@ -0,0 +1,89 @@
+= Cloud Topics
+:description: Cloud Topics are Kafka topics that enable users to trade off latency for lower costs.
+:page-beta: true
+// tag::single-source[]
+
+ifndef::env-cloud[]
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+endif::[]
+
+Starting in v25.3, Redpanda provides Cloud Topics to support multi-modal streaming workloads in the most cost-effective way possible. While standard Kafka xref:config-topics.adoc[topics] are ideal for latency-sensitive, high-throughput workloads (for example, for audit logs), Cloud Topics are optimized for high-throughput, cost-sensitive workloads that can tolerate higher latencies. Instead of replicating every byte across expensive network links, Cloud Topics leverage durable, inexpensive Cloud object storage (S3, GCS, MinIO) as the primary mechanism for backing up data. The difference in response time is typically 500ms, which is only impactful for latency-sensitive applications.
+
+== Prerequisites
+
+- xref:get-started:rpk-install.adoc[] v25.3 or later.
+ifndef::env-cloud[]
+- xref:manage:tiered-storage.adoc#set-up-tiered-storage[Set up] and xref:manage:tiered-storage.adoc#enable-tiered-storage[enable Tiered Storage] on your Redpanda cluster.
++
+[NOTE]
+====
+Be sure to set the following cluster-level properties to `false` to disable Tiered Storage for new topics:
```

Comment:
"While testing Cloud Topics, If you plan to use Cloud Topics for all topics in a redpanda cluster, be sure to set the following cluster-level properties to `false` to disable Tiered Storage for newly created topics, as Tiered Storage topics cannot currently be converted into cloud topics"

Reply:
@mattschumpert This is confusing. A prereq is that you enable TS, now here we tell them to disable it. Can you help me understand why we would offer up contradictory guidance wrt TS?